Sources/ScipioKit/Runner.swift

```diff
@@ -38,43 +38,92 @@ public struct Runner {
-            self.frameworkType = frameworkType
-            self.outputDirectory = outputDirectory
-            self.cacheMode = cacheMode
-            self.platformMatrix = platformMatrix
-            self.overwrite = overwrite
-            self.verbose = verbose
+        public struct BaseBuildOptions {
+            public var buildConfiguration: BuildConfiguration
+            public var platforms: PlatformSpecifier
+            public var isSimulatorSupported: Bool
+            public var isDebugSymbolsEmbedded: Bool
+            public var frameworkType: FrameworkType
+
+            public init(
+                buildConfiguration: BuildConfiguration = .release,
+                platforms: PlatformSpecifier = .manifest,
+                isSimulatorSupported: Bool = false,
+                isDebugSymbolsEmbedded: Bool = false,
+                frameworkType: FrameworkType = .dynamic
+            ) {
+                self.buildConfiguration = buildConfiguration
+                self.platforms = platforms
+                self.isSimulatorSupported = isSimulatorSupported
+                self.isDebugSymbolsEmbedded = isDebugSymbolsEmbedded
+                self.frameworkType = frameworkType
+            }
         }
+        public struct TargetBuildOptions {
+            public var buildConfiguration: BuildConfiguration?
+            public var platforms: PlatformSpecifier?
+            public var isSimulatorSupported: Bool?
+            public var isDebugSymbolsEmbedded: Bool?
+            public var frameworkType: FrameworkType?
 
-        public var buildConfiguration: BuildConfiguration
-        public var isSimulatorSupported: Bool
-        public var isDebugSymbolsEmbedded: Bool
-        public var frameworkType: FrameworkType
-        public var outputDirectory: URL?
-        public var cacheMode: CacheMode
-        public var platformMatrix: PlatformMatrix
-        public var overwrite: Bool
-        public var verbose: Bool
+            public init(
+                buildConfiguration: BuildConfiguration? = nil,
+                platforms: PlatformSpecifier? = nil,
+                isSimulatorSupported: Bool? = nil,
+                isDebugSymbolsEmbedded: Bool? = nil,
+                frameworkType: FrameworkType? = nil
+            ) {
+                self.buildConfiguration = buildConfiguration
+                self.platforms = platforms
+                self.isSimulatorSupported = isSimulatorSupported
+                self.isDebugSymbolsEmbedded = isDebugSymbolsEmbedded
+                self.frameworkType = frameworkType
+            }
+        }
 
         public enum CacheMode {
             case disabled
             case project
             case storage(any CacheStorage)
         }
+        public enum PlatformSpecifier: Equatable {
+            case manifest
+            case specific(Set<Platform>)
+        }
+
+        public enum Platform: String, Hashable {
```

Comment:
I introduce `Platform`. It represents destination platforms without Simulators. it's used for the user-facing interface.

Runner converts `Platform` to `SDK` by whether simulators should be included.
`SDK` will be used for building.